pkg/auth/routes.go

```diff
@@ -58,6 +58,7 @@ func NewRouter(p *deps.RootProvider, configSource *configsource.ConfigSource) *h
 		httproute.MiddlewareFunc(httputil.StaticSecurityHeaders),
 		RequestMiddleware(p, configSource, newRequestMiddleware),
 		p.Middleware(newUIParamMiddleware),
+		p.Middleware(newContextHolderMiddleware),
```

Comment:
This should be put above newPanicMiddleware

Reply:
Per offline discussion, only need to above `newPanicWebAppMiddleware` but not `newPanicMiddleware `